tests/README.md

```diff
@@ -71,4 +72,4 @@ sleep_to_meet_frame_rate = false # If true, slow the tick rate to match the movi
 
 `fscommand("quit")` is enabled for tests, and will end the test at the end of this frame or tick.
 
-You can use this to end a test prematurely before the set number of iterations elapses, which may be useful for timer tests.
\ No newline at end of file
+You can use this to end a test prematurely before the set number of iterations elapses, which may be useful for timer tests.
```

Comment:
Huh, what is this change? A line ending?

Reply:
This change adds a missing newline at the end of file, but GitHub somehow cannot show it properly :thinking: